ros/src/twist_controller/launch/dbw_sim.launch

```diff
@@ -8,7 +8,7 @@
         <param name="accel_limit" value="1." />
         <param name="wheel_radius" value="0.335" />
         <param name="wheel_base" value="3" />
-        <param name="steer_ratio" value="2.67" />
+        <param name="steer_ratio" value="14.8" />  <!-- 2.67 -->
```

Comment:
Where did you get that number from? How do you know that Carla's steer_ratio will be 14.8?

Reply:
The launch file for Carla is `ros/src/twist_controller/launch/dbw.launch`, which has that value for the steer ratio